extensions/brew/package.json

```
@@ -56,6 +56,14 @@
       "required": false,
       "title": "Custom Brew Executable Path",
       "description": "Set this if your brew executable is in a non-standard location"
+    },
+    {
+      "name": "zapCask",
+      "type": "checkbox",
+      "required": false,
+      "title": "Zap Casks",
+      "label": "Remove all files associated with a cask.",
+      "description": "Run cask uninstall with the --zap option for a more complete uninstallation of files associated with cask. WARNING: This may remove files shared with other applications."
```

I'd suggest adding a newline before the warning and removing the all caps.

---

```suggestion
      "description": "Run cask uninstall with the --zap option for a more complete uninstallation of files associated with cask.\nWARNING: This may remove files shared with other applications."
```
Oh I didn't realise you could add new lines to tooltips. I'll make sure that's done.

I think maybe its better to keep it in caps just have a higher chance of people noticing it? If you disagree feel free to make it commit the change, make a suggested change or just lmk and it'll be done.